tools/deployment/k8s/s3/deploy.sh

```
@@ -0,0 +1,92 @@
+echo "Downloading YScope log-viewer release from"
+echo ${RELEASE_URL}
+mkdir -p "$DECOMPRESSED_ASSETS_DIRECTORY"
+curl -sSL "$RELEASE_URL" | tar --strip-components 1 -xz -C "$DECOMPRESSED_ASSETS_DIRECTORY"
+
+# Wait until S3 endpoint is available
+echo "------------------------------------------------------------------------------------------"
+wait_for_s3_availability
+
+# Create log-viewer bucket if not already exist
+aws s3api head-bucket --endpoint-url "${AWS_ENDPOINT_URL}" --bucket "${LOG_VIEWER_BUCKET}" 2>/dev/null
+if [ $? -ne 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if ! mycmd;', not indirectly with $?.

(SC2181)